static/js/components/EducationForm.js

```diff
@@ -138,7 +189,9 @@ class EducationForm extends ProfileFormFields {
           <Cell col={7} />
           <Cell col={1}>
             <Switch
-              onChange={()=>{this.handleSwitchClick(level.value);}}
+              id={`profile-tab-education-switch-${level.value}`}
+              onClick={()=>{this.handleSwitchClick(level.value);}}
+              readOnly={true}
```

Comment:
I probably should have just explained this, haha.

I set an `onClick` handler because it greatly simplifies testing and the user experience is the same - I had a lot of difficulty getting the change event to fire, but simulating a click was relatively straightforward. If we don't set `onChange` then the component becomes a controlled component, so it doesn't have any value that changes on itself, but instead just broadcasts changes to the store and subscribes to updates.

Does that make sense? It might be that there's an easy way to get the `onChange` callback to fire when testing, I spent a lot time puzzling over it and didn't get it work :(